optuna/storages/rdb/storage.py

```diff
@@ -559,10 +559,12 @@ def _merge_trials_orm(
         for trial_id, trial in id_to_trial.items():
             params = {}
             params_in_internal_repr = {}
+            param_distributions = {}
```

Comment:
This file has imported `distributions` module. To avoid name conflict, I use a bit longer variable name here.